config/environments/development.rb

```diff
@@ -49,7 +49,7 @@
     end
 
   # Print deprecation notices to the Rails logger.
-  config.active_support.deprecation = :log
+  config.active_support.deprecation = :notify
```

Comment:
I think we don't want to keep this in development, do we?

Reply:
My bad! 😥